### DISCUSSION TITLE
Welcome to flexo Discussions!

### DISCUSSION THREAD

**Discussion by nroi** [nroi/flexo#101] [Announcements]
## 👋 Welcome!
  This is the right place to:
  * Ask questions about flexo.
  * Share ideas on how to improve flexo.
  * Engage with other community members.

**Comment by pidario** [comment]
Thanks for opening up the discussion!
It's not actually a new idea but I believe one of the first issue to tackle down should be #82. I've been using flexo for some time and this is the only thing that kinda ruin the experience, at least in my opinion.

**Reply by nroi** [reply]
Thanks for your feedback, @pidario 

In general, if one particular issue is important to you, what you can do is:
* Leave a thumbs up :+1:  at this issue.
* Describe why this issue is important to you.

**Reply by pidario** [reply]
Indeed, thanks anyway for writing a draft.
